diff --git a/node_modules/debug/src/node.js b/node_modules/debug/src/node.js
index 79bc085..8ffb809 100644
--- a/node_modules/debug/src/node.js
+++ b/node_modules/debug/src/node.js
@@ -155,7 +155,7 @@ exports.inspectOpts = Object.keys(process.env).filter(key => {
 function useColors() {
 	return 'colors' in exports.inspectOpts ?
 		Boolean(exports.inspectOpts.colors) :
-		tty.isatty(process.stderr.fd);
+		false;
 }
 
 /**
